test/test_transforms.py

```diff
@@ -205,6 +205,76 @@ def test_add_token(self):
     def test_add_token_jit(self):
         self._add_token(test_scripting=True)
 
+    def _pad_transform(self, test_scripting):
```

Comment:
Can you add a brief description of what this test does?
In the form of "(under some condition), an API produces a result that satisfies this property"